net-mail/vacation/vacation-1.2.7.0-r1.ebuild

```diff
@@ -17,25 +17,19 @@ RDEPEND="virtual/mta
 DEPEND="${RDEPEND}
 	!mail-mta/sendmail"
 
-src_unpack() {
-	unpack ${A}
-	cd "${S}"
-}
-
 src_prepare() {
-	eapply_user
+	default
 
 	sed -i -e "s:install -s -m:install -m:" Makefile || die "sed failed!"
 	sed -i -e "s:-Xlinker:${LDFLAGS} -Xlinker:" Makefile || die "sed failed!"
 }
 
 src_compile () {
-	emake CC=$(tc-getCC) ARCH=$(tc-arch-kernel) CFLAGS="${CFLAGS} -DMAIN" || die "emake failed."
+	emake CC=$(tc-getCC) ARCH=$(tc-arch-kernel) CFLAGS="${CFLAGS} -DMAIN"
 }
 
 src_install () {
 	dodir /usr/bin
```

Comment:
why do you need this?

Reply:
by request from ConiKost
https://github.com/gentoo/gentoo/pull/13499#discussion_r340735966

Reply:
Well I just took over editing this ebuild (to keep it in the ebuild db).
I don't know the exact reason why the additional variables are needed.
But it works well at my place

Reply:
I just gave it a try, CC and ARCH are not needed.
But the MAIN definition is needed, there are references in rfc822.c

I corrected the make call